pkg/controller/database/databaseHelper.go

```diff
@@ -38,14 +39,16 @@ func determinDatabaseType(dbcr *kciv1alpha1.Database, dbCred database.Credential
 
 	switch engine {
 	case "postgres":
+		extList := append(dbcr.Spec.Extensions, "pg_stat_statements") // enable pg_stat_statements extension by default
```

Comment:
This should maybe an optional choice. Maybe some people don't need or don't want this enabled (performance reasons) for a specific database.